frontend/src/v5/ui/controls/formTextArea/formTextArea.component.tsx

```diff
@@ -17,8 +17,7 @@
 
 import { FormTextField, FormTextFieldProps } from '@controls/formTextField/formTextField.component';
 
-export type FormTextAreaProps = FormTextFieldProps & {
-	control: any,
+export type FormTextAreaProps = Omit<FormTextFieldProps, 'variant'> & {
 	formError?: any,
```

Comment:
I think FormTextFieldProps already containers formError, so that can be removed here